central/complianceoperator/v2/report/manager/manager_impl.go

```diff
@@ -443,7 +453,7 @@ func (m *managerImpl) HandleResult(sensorCtx context.Context, result *storage.Co
 		return err
 	}
 	log.Debugf("Searching for ScanWatcher %s to PushCheckResult", id)
-	return m.getWatcher(sensorCtx, id).PushCheckResult(result)
+	return m.getWatcher(sensorCtx, id, 0).PushCheckResult(result)
```

Comment:
Same here. We should split.

Reply:
Done in [4ff5709](https://github.com/stackrox/stackrox/pull/15517/commits/4ff5709d89ba20bc91a10aa2300afcce419fce54)